src/components/Carousel/index.js

```diff
@@ -16,27 +16,27 @@ class Arrow extends PureComponent {
     children: PropTypes.element,
     className: PropTypes.string,
     direction: PropTypes.oneOf(['left', 'right']).isRequired,
```

Comment:
Should this be updated to `prev` and `next` to match the nomenclature used below for `handlePress`?

Reply:
👍